src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -39,13 +42,12 @@
             "WHERE (c.employee.employeeId = :employeeId AND " +
             "(c.startDate BETWEEN :startDate AND :today) OR " +
             "(c.endDate BETWEEN :today AND :endDate) OR " +
-            "(c.startDate < :startDate AND (c.endDate IS NULL OR :endDate > :today)))" +
-            ")"
-    )
+            "(c.startDate < :startDate AND (c.endDate IS NULL OR :endDate > :today))))")
```

Comment:
@eoinmcafee you'd prefer today be passed through like this rather than be derived from the SQL itself (ie using something like: current_date() found in http://www.coding-dude.com/wp/java/hql-date-datetime-quick-tip/ )